contributing.md

```diff
@@ -0,0 +1,136 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> Our website runs off of VuePress. Here you will find contribution resources and advice.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+## Preparing the development environment
+### Fork the project
+* Create your own fork of the repository by choosing the **Fork** option at the top right of the website github repository.
+* Create a new branch before working on your new fork. The method of branching depends on the Git CLI/GUI that you use, reference your software manual, or contact us if you need help with this step.
+* Once on your own branch, you're ready to start making improvements to the website
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request. This can be done by navigating to the main page of the [website git repository](https://github.com/Paperback-iOS/website) and selecting 'pull request' at the top.  You may have to click the button titled **compare across forks**. Ensure that the base repository belongs to Paperback-iOS, whereas the head is your fork, and your branch.
+
+### Running the website
+> When you make modifications to the site, you should validate your changes and formatting yourself first.
+
+See [Building local enviroment](https://github.com/Paperback-iOS/website#building-local-enviroment)
+
+## Guidelines
+### Glossary
+
+ * App/application --> app
+ * Device/iPhone/iPad/phone --> iPhone or iPad
+ * iOS --> iOS or iPadOS
```

Comment:
I think this will look nicer in a table

```suggestion
| Variations               | To be used    |
|:------------------------:|:-------------:|
| App/application          | app           |
| Device/iPhone/iPad/phone | iOS or iPadOS |
| iOS                      | iOS or iPadOS |
```